string key      # Key of the parameter to be found
---
string service  # Name of the parameter service, if a parameter with the requested key could be found
